keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.keel.clouddriver
+
+import com.netflix.spinnaker.keel.clouddriver.model.Network
+import com.netflix.spinnaker.keel.clouddriver.model.SecurityGroup
+import org.springframework.stereotype.Component
+
+@Component
+class MemoryCloudDriverCache(
```

Comment:
Not actually doing any cacheing yet.

Reply:
Why throw instead of nullable?

Reply:
I figured any time we're using this we expect the values to be legit. Certainly when converting from CD model -> Keel. I guess not necessarily the other way but then we'd want to abort the intent.